.travis.yml

```diff
@@ -25,6 +25,24 @@ matrix:
           packages: [ 'libstdc++6', 'libstdc++-5-dev' ]
     - os: osx
       osx_image: xcode8.3
+    # clang-tidy specific job
+    - os: linux
+      sudo: false
+      addons:
+        apt:
+          sources: [ 'ubuntu-toolchain-r-test' ]
+          packages: [ 'libstdc++6', 'libstdc++-5-dev' ]
```

Comment:
This is just fine, but `libstdc++-5-dev` does depend on `libstdc++6` so the latter will get pulled in automatically and therefore the behavior is the same if it is left off and only `libstdc++-5-dev` is used.

Reply:
@springmeyer Cool, so update this for the other Travis jobs as well? They are all doing the same.

Reply:
Yes, cool to use `libstdc++-5-dev` for all @GretaCB